spec/rails_helper.rb

```diff
@@ -31,6 +31,7 @@
 Capybara.server = :puma
 Capybara.javascript_driver = :selenium_chrome_headless
 Capybara.enable_aria_label = true
+Capybara.disable_animation = true
```

Comment:
This should make the weird timing problems in admin/action_creation_spec go away. I didnt remove everything in there but it should remove the need for sleeps.